src/web/app/pages-help/instructor-help-page/instructor-help-page.component.html

```diff
@@ -5,7 +5,7 @@ <h1 class="color-orange"> Help for Instructors</h1>
       <div>
         <p>
           Have questions about how to use TEAMMATES? This page answers some frequently asked questions.
-          <br> If you are new to TEAMMATES, our <a routerLink="../getting-started">Getting Started</a> guide will introduce you to the basic functions of TEAMMATES.
+          <br> If you are new to TEAMMATES, our <a [routerLink]="instructorGettingStartedPath">Getting Started</a> guide will introduce you to the basic functions of TEAMMATES.
```

Comment:
Any advantage of doing this instead of `/web/front/help/getting-started`?

Reply:
You assumed `/web/front/help/getting-started` is the only possible value here, which is wrong